packages/truffle-debugger/lib/controller/sagas/index.js

```diff
@@ -14,8 +14,11 @@ import * as actions from "../actions";
 
 import controller from "../selectors";
 
+//NOTE: when updating this don't forget to update CONTROL_ACTIONS in
+//reducers.js as well!
 const CONTROL_SAGAS = {
   [actions.ADVANCE]: advance,
+  [actions.MULTIADVANCE]: multiadvance,
```

Comment:
Here's an idea: what do you think about making `actions.ADVANCE` just optionally take an argument for the number of steps to advance? This way we won't need the extra action/saga/etc.

Reply:
I thought of that, but I liked having `advance` as a separate more primitive thing.  It would throw off some of the correspondence, at least, to do that.  Like, all the other existing sagas that currently call `advance` -- would they still call `advance`, but with an argument?  Would they directly call `trace.advance`?  Would what's currently `advance` be renamed to something else?  I decided to leave it alone and just add it as something separate.

Reply:
> would they still call `advance`, but with an argument?

Default values for arguments would solve this concern.

I see the argument for having a separate action, but I'm not sure that `advance(n)` is any less primitive than `advance(1)`? At least in the controller.

Anyway my opinion isn't too strong here; I just think it's cleaner to modify `advance` rather than make a new action. But I leave it up to you to decide what you prefer.

Reply:
Oh hm -- yeah doing it with a default argument would make sense.  OK, I think I'll change it to that way after all.